src/xdsl/dialects/std.py

```diff
@@ -116,6 +70,12 @@ def verify_(self) -> None:
         if self.input1.typ != self.input2.typ or self.input2.typ != self.output.typ:
             raise Exception("expect all input and output types to be equal")
 
+    @staticmethod
+    def get(operand1: Union[Operation, SSAValue],
+            operand2: Union[Operation, SSAValue]) -> Addi:
+        return Addi.build(operands=[operand1, operand2],
+                          result_types=[IntegerType.from_width(32)])
```

Comment:
Do MLIR addis only work on 32bit integers? Maybe add a size argument that has default.

Also, i jus t looked at the MLIR docu, is it possible that simple arithmetic got taken out of `std` into `arith`? Maybe we should do the same (or rename them to `arith.` instead of `std.`).

Reply:
The type should be a parameter as will, I think. 
MLIR is currently refactoring this, so yes, we should probably introduce a `arith` dialect at some point.

Reply:
Maybe we can make the `result_type` an optional parameter. 32 bit will be the one type we use most frequently for now, but later on it would be nice to have this flexibility. 
On the other hand, one has still access to the `Addi.build` factory.

Reply:
Yeah, I kept the operation as it was before, but I think we should overall make an additional pass later on the dialects. For example, affine is not up to date at all, and here std is a bit broken.